ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/WatchedImages/WatchedImagesModal.tsx

```diff
@@ -21,53 +33,122 @@ function WatchedImagesModal({
                     </Alert>
                 )}
-                {isError && (
+                {watchImageMutation.isError && (
                     <Alert
                         variant="danger"
                         isInline
                         title="There was an error adding the image to the watch list"
                     >
-                        {getAxiosErrorMessage(error)}
+                        {getAxiosErrorMessage(watchImageMutation.error)}
                     </Alert>
                 )}
                 <WatchedImagesForm
                     defaultWatchedImageName={defaultWatchedImageName}
-                    watchImage={mutate}
+                    watchImage={watchImageMutation.mutate}
+                    watchedImagesRequest={currentWatchedImagesRequest}
                 />
+                <Divider component="div" />
+                {unwatchImageMutation.isSuccess && (
+                    <Alert
+                        variant="success"
+                        isInline
+                        title="The image was successfully removed from the watch list"
+                    />
+                )}
+                {unwatchImageMutation.isError && (
+                    <Alert
+                        variant="danger"
+                        isInline
+                        title="There was an error removing the image from the watch list"
+                    >
+                        {getAxiosErrorMessage(unwatchImageMutation.error)}
+                    </Alert>
+                )}
+                {currentWatchedImagesRequest.error && (
+                    <Alert
+                        variant="danger"
+                        isInline
+                        title="There was an error loading the current list of watched images"
+                    >
+                        {getAxiosErrorMessage(currentWatchedImagesRequest.error)}
+                    </Alert>
+                )}
+                {currentWatchedImagesRequest.loading && !currentWatchedImagesRequest.data && (
+                    // Display a loading spinner
+                    <Bullseye>
+                        <Spinner isSVG aria-label="Loading current watched images" />
+                    </Bullseye>
+                )}
+                {currentWatchedImagesRequest.data && (
+                    <>
+                        <Title id="current-watched-images-list" headingLevel="h2">
+                            {pluralize(watchedImages.length, 'watched image')}
+                        </Title>
+                        <WatchedImagesTable
+                            aria-labelledby="current-watched-images-list"
+                            className="pf-u-max-height"
+                            style={
+                                {
+                                    overflowY: 'auto',
+                                    '--pf-u-max-height--MaxHeight': '280px',
```

Comment:
This increases the 200px from VM 1.0 - this is the highest number that doesn't result in a scrollbar default on the little 13" MacBook display.